libyul/Utilities.cpp

```diff
@@ -59,6 +59,13 @@ string solidity::yul::reindent(string const& _code)
 	for (string& line: lines)
 		boost::trim(line);
 
+	// Reduce multiple consecutive empty lines.
+	lines = fold(lines, vector<string>{}, [](auto&& _lines, auto&& _line) {
+		if (!(_line.empty() && !_lines.empty() && _lines.back().empty()))
+			_lines.emplace_back(std::move(_line));
+		return _lines;
```

Comment:
Yea I guess it needs `move(_lines)` here

Reply:
ah, maybe!